docs/detections/rules-ui-manage.asciidoc

```diff
@@ -104,6 +105,22 @@ NOTE: When duplicating a rule with exceptions, you can choose to duplicate the r
 * To enable or disable a single rule, switch on the rule's *Enabled* toggle.
 * To <<snooze-rule-actions,snooze>> actions for rules, click the bell icon.
 
+[float]
+[[manually-run-rules]]
+=== Manually run rules
+
+Manually run enabled rules during a specfied time range. 
+
+. Go to **Rules** -> **Detection rules (SIEM)**.
+. Do one of the following:
+* Select the **All actions** menu (**...**) on a rule, then select **Manual run**.
+* Select a rule to open its details page, select the **All actions** menu, then select **Manual run**.
+* Select all the rules you want to manually run, select the **Bulk actions** menu, then select **Manual run**.
+. Specify a time range for the manual run. By default, the time range is set to the current day and three hours in the past.
```

Comment:
It's not clear to me what this range is for. At first I thought maybe the rule runs during the range, but I don't know how that'd work if it defaults to the past. Is it looking at the past 3 hours of event data? I thought that's what the rule's "look-back time" setting does?

Reply:
Yeah, this could be more precise. When users select the time range for the manual run, they're really choosing when the manual run starts and when it ends. How's this:

```suggestion
. Specify when the manual run starts and ends, or keep the default selection, which is the current day and three hours in the past. The rule will search for events during the selected time range.
```

Reply:
Clearer now, thanks! The first sentence is a little busy now, maybe break it up:
```suggestion
. Specify when the manual run starts and ends. The default selection is the current day starting three hours in the past. The rule will search for events during the selected time range.
```